crates/e2e-test-utils/src/transaction.rs

```diff
@@ -16,7 +16,17 @@ pub struct TransactionTestContext;
 impl TransactionTestContext {
     /// Creates a static transfer and signs it, returning an envelope.
     pub async fn transfer_tx(chain_id: u64, wallet: PrivateKeySigner) -> TxEnvelope {
-        let tx = tx(chain_id, 21000, None, None, 0);
+        let tx = tx(chain_id, 21000, None, None, 0, Some(20e9 as u128));
```

Comment:
it'd be great if we could improve ergonomics for this a bit, so that the user can provide TransactionRequest